oeps/oep-0011-bp-FED-technology.rst

```diff
@@ -48,33 +48,42 @@ Specification
 Technology Selection
 ~~~~~~~~~~~~~~~~~~~~
 
-1. **Server-side content should be rendered with Django Templates**
```

Comment:
@georgebabey Can you update the following fields in the header of this OEP:

- Author - add yourself
- Last Modified (we may eventually remove this field since it's already captured in Git, but for now let's keep it updated)
- Status - change to "Accepted" (they may have forgotten to do this last time)